lightningd/datastore.c

```diff
@@ -343,6 +349,11 @@ static struct command_result *json_datastoreusage(struct command *cmd,
 	     stmt;
 	     stmt = wallet_datastore_next(cmd, cmd->ld->wallet, stmt, 
 	     				  &k, &data, &gen)) {
+
+		// Break if we got a key that we are not interested in.
```

Comment:
Just merged API change means you don't need to do this any more: wallet_datastore_next will do it.

Reply:
Rebased to include the API change.